setup/install-klayout.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh
 
+# Get directory of script
+src_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
+
 mkdir -p deps
 cd deps
 
-pkg_version="0.27.8-1"
+pkg_version=$(python3 ${src_path}/_tools.py --tool klayout --field version)
```

Comment:
My gut says that sed or awk might be better for extracting strings in a bash script, but JSON isn't fun to parse that way, and this *is* a Python project. Who knows, maybe someday we'll move these setup scripts from .sh to .py.

Reply:
sed or awk would be pretty hard to manage. The point of the _tools.json file is to have the versions in a single place and the python script here helps get the information out. We could look at making python scripts, but given how finicky these install scripts are to begin with I don't know if we want to be the single source of install scripts.